src/Jobs/Recruit.Vacancies.Jobs/VacancyStatus/VacancyStatusJob.cs

```diff
@@ -15,16 +16,24 @@ public class VacancyStatusJob
         private const bool CanRunOnStartup = false;
 #endif
```

Comment:
Do we need the compiler directives now?

https://github.com/SkillsFundingAgency/das-recruit/blob/00d9fbb45c6697e4a3a33a36d33e671e0c336001/src/Jobs/Recruit.Vacancies.Jobs/VacancyStatus/VacancyStatusJob.cs#L13-L17